assets/shells/bashrc.sh

```diff
@@ -21,7 +21,10 @@ alias {{.ExecName}}='{{.ExecAlias}}'
 {{ end }}
 
 {{range $K, $CMD := .Scripts}}
-alias {{$K}}='{{$.ExecName}} run {{$CMD}}'
+function {{$K}} {
+    {{$.ExecName}} run {{$CMD}} "$@"
+}
+export -f {{$K}}
```

Comment:
I don't think we need this `export`, given this code will effectively be injected into your bashrc

Reply:
.bashrc isn't sourced for things like `bash -c` or scripts that start with `#!/bin/bash` so the export makes the commands available in those situations

Reply:
the environment inherited is just the environment variables which *nix operating systems understand at a lower level (see `execle()`'s `char *const envp[]` argument for example).  It doesn't include shell-specific items like aliases and functions.  `export -f` is a special variant in bash which tells it to copy functions from the current shell into spawned _bash_ shells specifically.   See `man bash` for details on how `export -f` works